pubspec.yaml

```diff
@@ -4,22 +4,22 @@ repository: https://github.com/mjablecnik/MyLogger
 version: 1.0.1
 
 environment:
-  sdk: '>=2.14.0 <4.0.0'
```

Comment:
Is some important purpose why did you removed Dart version support older than 3.0.6?

Reply:
No, I just propose to use modern dart SDK as it's stable enough. This can be reverted if you think it would increase lib sharing area.

Reply:
I would not remove old Dart version support until if it is really necessary. 
For example if somebody will want to add some new functionality requiring some new SDK features..